### DISCUSSION TITLE
Transmitter Error

### DISCUSSION THREAD

**Discussion by GLoren1** [NightscoutFoundation/xDrip#2500] [General]
I replaced my transmitter after unsuccessfully trying to start an old transmitter after 100 days of use.  The new transmitter, which has a shelf life until 2023, will not connect.   I restarted the phone,  restarted the collector, cleared the data/cache, and restarted the phone again.  When that didn't work, I re-entered the transmitter ID and saved it  My android bluetooth settings does not show the transmitter as an available device

I've attached the status pages [](url)
![temp](https://user-images.githubusercontent.com/62320462/204077049-3909ca60-3087-4fe7-87f4-28b0ed42871b.jpg)
![temp2](https://user-images.githubusercontent.com/62320462/204077058-7236c8a4-d236-4701-adfd-561687765dd0.jpg)

**Comment by Navid200** [comment]
Looks like you are using an unofficial version of xDrip.
Is there a reason for that?
It could be anything in that being the cause.  Or, may be not.  But, supporting every possible edit people may have done to xDrip is not possible.

If xDrip is missing something you need, you can ask for it to be added in these same discussions.  There is no guarantee it will be added specially if there is a good reason it has not been added.  

Where is the transmitter exactly?
Are you sure you don't have a typo in the transmitter ID?

**Reply by GLoren1** [reply]
I installed this version to work with a Mi band before that functionality was integrated into xDrip.  I'm reluctant to upgrade because it was such a challenge to get Mi band to work with the version I'm using.  

The transmitter "error" was my vision error!  I couldn't read the transmitter ID correctly so entered a wrong number.!

**Reply by Navid200** [reply]
So, it's working now?

**Comment by GLoren1** [comment]
Yes. Thank you for your help.
